doc/conf.py

```diff
@@ -17,6 +17,11 @@
 import os
 from subprocess import check_output
 
+import sphinxcontrib.katex as katex
+
+sys.path.insert(0, os.path.abspath('.'))
+from math_definitions import latex_macros
```

Comment:
This is a little bit ugly, but we could also do the `latex_macros` definition directly in `conf.py`.